### ISSUE TITLE
Upstreaming PetriNet --> Presentation code

### ISSUE THREAD

**Issue by jpfairbanks** [AlgebraicJulia/AlgebraicRelations.jl#10]
For the workflow schema features you wrote some code for generating a Presentation of an SMC from a Petri Net, should that get upstreamed to AlgebraicPetri?

**Comment by bosonbaas** [comment]
Yes, that's the current plan! Currently we're actually just using Petri.jl for the graphing (no AlgebraicPetri dependency), so the plan is for this to be upstreamed to AlgebraicPetri once the other graphing utilities are transferred from Petri.jl.

**Comment by jpfairbanks** [comment]
Awesome, it would also be great to have a PetriNets to Presentation(SMC) too. This would come in handy on SocialTheories where I need to represent presentations of SMCs as data.
